templates/ts/src/react-app-env.d.ts

```diff
@@ -1,13 +1,15 @@
 /// <reference types="react-scripts" />
 
 declare module '*.css' {
-  const classes: { readonly [key: string]: string };
-  export default classes;
+  export default style as string
 }
 
 declare module '*.scss' {
-  const classes: { readonly [key: string]: string };
-  export default classes;
+  export default style as string
+}
+
+declare module '*.sass' {
+  export default style as string
 }
```

Comment:
the true value of the css imports are strings right now (before & after patch).
therefore we should change these types to reflect the true value to avoid confusion and compile errors